cmd/podman/varlink/io.podman.varlink

```diff
@@ -554,6 +554,10 @@ method GetContainer(id: string) -> (container: Container)
 # user environment, results might differ from what you expect.
 method GetContainersByContext(all: bool, latest: bool, args: []string) -> (containers: []string)
 
+# GetContainerByID allows you to get a container id from its partial or full name
+# or partial ID
+method GetContainerByID(id: string) -> (containers: string)
```

Comment:
This endpoint is a special case of GetContainersByContext(false, false, ['name']).  Is the specialization worth owning the endpoint forever?  Varlink doesn't have versioned API's so once released we own it. :-)

Reply:
The problem is that `shortcuts.GetContainersByContext()`  called by `GetContainersByContext()` may fail to return a proper `*Container` (that was the main reason to add the evict functionality). While retrieving an ID should not fail.

Not sure how we could do otherwise :thinking: